src/driver_client/dclient_cfg.rs

```diff
@@ -0,0 +1,47 @@
+pub enum CardType {
+    C1100,
+}
+
+/// The [`DriverConfig`] is a struct that defines a set of 64-bit unsigned integer (`u64`)
+/// representing addreses memory space for different components of a FPGA.
+/// The struct is divided into logical parts: AXI Lite space of addresses and AXI space of addresses
+#[derive(Copy, Clone, Debug)]
+pub struct DriverConfig {
+    // CTRL
+    pub(crate) ctrl_baseaddr: u64,
+    pub(crate) ctrl_cms_baseaddr: u64,
+    // pub(crate) ctrl_qspi_baseaddr: u64,
+    pub(crate) ctrl_hbicap_baseaddr: u64,
+    // pub(crate) ctrl_mgmt_ram_baseaddr: u64,
+    pub(crate) ctrl_firewall_baseaddr: u64,
+    pub(crate) dma_firewall_baseaddr: u64,
+    pub(crate) ctrl_dfx_decoupler_baseaddr: u64,
+
+    // DMA
+    pub(crate) dma_baseaddr: u64,
+    // pub(crate) dma_hbicap_baseaddr: u64,
+}
+
+impl DriverConfig {
+    /// Create a new driver config.
+    pub fn driver_client_cfg(card_type: CardType) -> Self {
+        match card_type {
+            CardType::C1100 => c1100_cfg(),
+        }
+    }
+}
+
+fn c1100_cfg() -> DriverConfig {
```

Comment:
We might want to change the naming here since this config is compatible with the U250 as well.